commitizen/cz/conventional_commits/conventional_commits.py

```diff
@@ -172,39 +176,47 @@ def message(self, answers: dict) -> str:
         return message
 
     def example(self) -> str:
-        return (
+        return self.config.settings.get(
+            "example",
             "fix: correct minor typos in code\n"
             "\n"
             "see the issue for details on the typos fixed\n"
             "\n"
-            "closes issue #12"
+            "closes issue #12",
```

Comment:
not sure why formatter insisted on comma here

Reply:
AFAIK, that's how black works. It prefer adding a following comma to avoid the error that you add another item without adding this comma in the future.